packages/@glimmer/core/test/interactive/modifier-test.ts

```diff
@@ -24,23 +70,276 @@ module('Modifier Tests', () => {
       )
     );
 
-    const element = document.getElementById('qunit-fixture')!;
-
-    await render(MyComponent, element);
     assert.strictEqual(
-      element.innerHTML,
+      await render(MyComponent),
       `<button>Count: 0</button>`,
       'the component was rendered'
     );
 
-    const button = element.querySelector('button')!;
-    button.click();
+    click('button');
 
-    await settled();
     assert.strictEqual(
-      element.innerHTML,
+      await settled(),
```

Comment:
Seems somewhat surprising that `settled` returns the HTML

Reply:
This is the pattern established by the `render()` test helper. I agree, the more I work with it the more I think we need something a bit better. Ideally we would get QUnit.dom working in this repo, and allow `render()` to return the result (along with a way to destroy the result).